src/components/delete-trace.tsx

```diff
@@ -78,3 +78,34 @@ export function DeleteTrace({
     </Popover>
   );
 }
+
+export function DeleteTraceMultiSelectAction({
+  traceIds,
+  projectId,
+}: {
+  traceIds: string[];
+  projectId: string;
+}) {
+  const utils = api.useUtils();
+
+  const hasAccess = useHasAccess({ projectId, scope: "traces:delete" });
+
+  const mutDeleteTraces = api.traces.deleteMany.useMutation({
+    onSuccess: () => void utils.traces.invalidate(),
+  });
+
+  if (!hasAccess) {
```

Comment:
let's show the button but disable it. Otherwise the menu is empty for users that lack permissions